src/System.CommandLine/Argument{T}.cs

```diff
@@ -210,10 +210,10 @@ void UnrecognizedArgumentError(ArgumentResult argumentResult)
         /// <returns>The configured argument.</returns>
         public Argument<T> AcceptLegalFilePathsOnly()
         {
-            var invalidPathChars = Path.GetInvalidPathChars();
```

Comment:
Wasn't this meant to cache `Path.GetInvalidPathChars()` which returns a new char[] on each call?

Reply:
What we had was:

1. calling `Path.GetInvalidPathChars()` for every `AcceptLegalFilePathsOnly` call
2. storing it's result in local variable (not a static field of the class, which could allow for caching the value)
3. allocating a closure (a lambda with a state)

What I am offering here is lack of closure allocation. I would prefer to avoid caching the result of `Path.GetInvalidPathChars()` in a static field as long as profiling don't prove that it's needed (adding static fields can hurt startup scenarios, it's not always a free lunch)